s3stream/src/main/java/com/automq/stream/s3/ByteBufAlloc.java

```diff
@@ -25,11 +25,16 @@
 import java.util.Optional;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.atomic.LongAdder;
+import io.netty.util.internal.StringUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class ByteBufAlloc {
-    public static final boolean MEMORY_USAGE_DETECT = Boolean.parseBoolean(System.getenv("AUTOMQ_MEMORY_USAGE_DETECT"));
+
+    public static final String AUTOMQ_MEMORY_USAGE_DETECT = "AUTOMQ_MEMORY_USAGE_DETECT";
+    public static final String AUTOMQ_MEMORY_USAGE_DETECT_INTERVAL = "AUTOMQ_MEMORY_USAGE_DETECT_TIME_INTERVAL";
+    public static final boolean MEMORY_USAGE_DETECT = Boolean.parseBoolean(System.getenv(AUTOMQ_MEMORY_USAGE_DETECT));
+    public static long memoryUsageDetectionInterval = 60000;
```

Comment:
You could use Systems.getEnvInt to get the environment config

Reply:
it's done.